src/OpenTelemetry.AutoInstrumentation/Configuration/EnvironmentConfigurationTracerHelper.cs

```diff
@@ -189,6 +190,16 @@ public static TracerProviderBuilder AddQuartzInstrumentation(TracerProviderBuild
                 .AddLegacySource("Quartz.Job.Veto");
         }
 
+#if NET6_0_OR_GREATER
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        public static TracerProviderBuilder AddEntityFrameworkCoreInstrumentation(TracerProviderBuilder builder, PluginManager pluginManager, LazyInstrumentationLoader lazyInstrumentationLoader)
+        {
+            lazyInstrumentationLoader.Add(new EntityFrameworkCoreInitializer(pluginManager));
+
+            return builder.AddSource("OpenTelemetry.EntityFrameworkCore");
```

Comment:
Next instrumentation library version will change it to `OpenTelemetry.Instrumenatation.EntityFrameworkCore`.
It can be merged as is, or we cant wait some time with newer release.

Reply:
We can merge it as it is and bump once it is released.